kura/distrib/config/kura.build.properties

```diff
@@ -36,7 +36,7 @@ org.eclipse.kura.driver.ble.sensortag.provider.version=1.0.200
 org.eclipse.kura.driver.eddystone.provider.version=1.0.0
 org.eclipse.kura.driver.ibeacon.provider.version=1.0.0
 org.eclipse.kura.driver.gpio.provider.version=1.0.100
-org.eclipse.kura.linux.bluetooth.version=1.0.400
+org.eclipse.kura.linux.bluetooth.version=1.0.500-SNAPSHOT
 org.eclipse.kura.linux.clock.version=1.0.400
```

Comment:
Please update also the versions of the beacon, ibeacon and eddystone examples.

Reply:
The versions were updated and will be pushed shortly.